src/script/page/MainContent/panels/preferences/accountPreferences/DataUsageSection.tsx

```diff
@@ -43,41 +40,37 @@ interface DataUsageSectionProps {
   teamState?: TeamState;
 }
 
-const DataUsageSection: React.FC<DataUsageSectionProps> = ({
-  propertiesRepository,
-  brandName,
-  isActivatedAccount,
-  teamState = container.resolve(TeamState),
-}) => {
+const DataUsageSection: React.FC<DataUsageSectionProps> = ({propertiesRepository, brandName, isActivatedAccount}) => {
```

Comment:
We trying to not to use React.FC :)
```suggestion
const DataUsageSection = ({propertiesRepository, brandName, isActivatedAccount}: DataUsageSectionProps) => {
```